Wishlist dropin (#347)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/index.mdx

---
title: Wishlist overview
description: Learn about the purpose, features, and use cases for the Wishlist drop-in component.
---

import { Badge } from '@astrojs/starlight/components';

> The Wishlist drop-in component provides to both guests and registered customers with a mechanism to store

Reviewer: ```suggestion
The Wishlist drop-in component provides both guests and registered customers with a mechanism to store products they are interested in purchasing later.
```

> products they are interested in purchasing at a later time. The Wishlist component is fully customizable

Reviewer: ```suggestion
The `Wishlist` component is fully customizable and can be integrated into your storefront to enhance the shopping experience.
```

> and can be integrated into your Adobe Commerce storefront to enhance the shopping experience.

Reviewer: ```suggestion
```

## Use cases

> The Wishlist drop-in component provides a variety of fully-customizable controls to showcase your products

Reviewer: ```suggestion
The `Wishlist` drop-in component offers a variety of fully-customizable controls to highlight your products (according to your brand's aesthetics) and create interactive experiences.
```

> (according to your brand's aesthetics) and build interactive experiences that engage customers. Use cases

Reviewer: ```suggestion
```

> include:

Reviewer: ```suggestion
Use cases include:
```

> - **Buy it later**: A user has found an interesting product and wants to keep it in a place that allows to quickly start the purchase process later.

Reviewer: ```suggestion
- **Buy it later**: A user finds an interesting product and wants to keep it in a place where they can quickly purchase it later.
```

> - **Synchronization**: A guest user wishlists a product and once he / she logs in, that product can be seen from other devices in which the user is also logged in.

Reviewer: ```suggestion
- **Synchronization**: Once a guest user logs in, the product they wish to purchase can be seen on other devices where the user is also logged in.
```
